JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/utils/ModifierUtils.kt

```diff
@@ -78,6 +83,39 @@ fun Modifier.handleDPadKeyEvents(
     false
 }
 
+/**
+ * Handles all D-Pad Keys
+ * */
+fun Modifier.handleDPadKeyEvents(
+    onLeft: (() -> Unit)? = null,
+    onRight: (() -> Unit)? = null,
+    onUp: (() -> Unit)? = null,
+    onDown: (() -> Unit)? = null,
+    onEnter: (() -> Unit)? = null
+) = onKeyEvent {
+
+    if (DPadEventsKeyCodes.contains(it.nativeKeyEvent.keyCode) && it.nativeKeyEvent.action == KeyEvent.ACTION_UP) {
```

Comment:
I think the `DPadEventsKeyCodes.contains(...)` is redundant and a bit dangerous.

You can easily add another when case and forget to add the same event to the `DPadEventsKeyCodes` and have nice time figuring out why it doesn't work 😄 .

The same working solution would be just to have `else -> return@onKeyEvent false` ?